dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/timer/CitationCountUpdateTimerInfo.java

```diff
@@ -0,0 +1,24 @@
+package edu.harvard.iq.dataverse.timer;
+
+import java.io.Serializable;
+
+public class CitationCountUpdateTimerInfo implements Serializable {
+
+    String serverId;
```

Comment:
Is this needed to be package-private instead of private?

Reply:
as above